utils_cv/detection/model.py

```diff
@@ -109,9 +149,8 @@ def __init__(self, dataset: Dataset, model: nn.Module = None):
 
         # setup model, default to fasterrcnn
         if self.model is None:
-            self.model = get_pretrained_fasterrcnn(len(dataset.labels)).to(
-                self.device
-            )
+            self.model = get_pretrained_fasterrcnn(len(dataset.labels))
+        self.model.to(self.device)
```

Comment:
Move this out of the IF clause to make sure the model always gets send to the correct device

Reply:
cool good call